### PR TITLE
Remove the word 'manager'

### PR DESCRIPTION
**Pull request by MrMusic**
Based on a suggestion by @AlexanderSupp, the word 'Manager' should be removed from the backend menu items and the same terms should be used as in Joomla itself.

### CODE REVIEW

**Comment by AlexanderSupp** [comment]
I installed “JG4-dev-Remove-the-word-manager-from-the-menu-items" and it looks fine.
I didn't uninstall JG4 before.
Based on the terrible test environment, I lose the filed "Tags" in Images now.
Maybe we discuss our test procedure in our next meeting.
Is it possible to establish a separate test stage in github between "main" and the "Pull requests".

**Comment by MrMusic** [comment]
@AlexanderSupp For me i use different Joomla installations to test different pull requests.
It may seem complicated but it is the only way (for me) to have a clean environment and avoid side effects.

**Comment by eumel1602** [comment]
I don't really care. I will then adjust the German translation after the decision...

File: administrator/com_joomgallery/language/en-GB/com_joomgallery.ini [hunk 1]
```diff
@@ -14,12 +14,10 @@ JOOMGALLERY="JoomGallery"
...
 COM_JOOMGALLERY_CONFIGURATION="JoomGallery: Options"
 COM_JOOMGALLERY_HELP="Help and Information"
-COM_JOOMGALLERY_IMAGE_MANAGER="Image Manager"
-COM_JOOMGALLERY_CATEGORY_MANAGER="Category Manager"
-COM_JOOMGALLERY_TAGS_MANAGER="Tags Manager"
-COM_JOOMGALLERY_CONFIGURATION_MANAGER="Configuration Manager"
-COM_JOOMGALLERY_MAINTENANCE_MANAGER="Maintenance Manager"
-COM_JOOMGALLERY_MIGRATION_MANAGER="Migration Manager"
+COM_JOOMGALLERY_CATEGORIES="Categories"
+COM_JOOMGALLERY_CONFIGURATIONS="Configurations"
```
[the remark this answers is not in the document]
Comment (Elfangor93) [reply administrator/com_joomgallery/language/en-GB/com_joomgallery.ini:18]: The constants "COM_JOOMGALLERY_CONFIG_SET" and "COM_JOOMGALLERY_CONFIG_SETS" are already existing and used. ;-)

File: administrator/com_joomgallery/language/en-GB/com_joomgallery.ini [hunk 2]
```diff
@@ -14,12 +14,10 @@ JOOMGALLERY="JoomGallery"
 COM_JOOMGALLERY="JoomGallery"
 COM_JOOMGALLERY_CONFIGURATION="JoomGallery: Options"
 COM_JOOMGALLERY_HELP="Help and Information"
-COM_JOOMGALLERY_IMAGE_MANAGER="Image Manager"
-COM_JOOMGALLERY_CATEGORY_MANAGER="Category Manager"
-COM_JOOMGALLERY_TAGS_MANAGER="Tags Manager"
-COM_JOOMGALLERY_CONFIGURATION_MANAGER="Configuration Manager"
-COM_JOOMGALLERY_MAINTENANCE_MANAGER="Maintenance Manager"
-COM_JOOMGALLERY_MIGRATION_MANAGER="Migration Manager"
+COM_JOOMGALLERY_CATEGORIES="Categories"
```
[the remark this answers is not in the document]
Comment (MrMusic) [reply administrator/com_joomgallery/language/en-GB/com_joomgallery.ini]: Agreed.

File: administrator/com_joomgallery/language/en-GB/com_joomgallery.ini (hunk 1 above)
[the remark this answers is not in the document]
Comment (MrMusic) [reply administrator/com_joomgallery/language/en-GB/com_joomgallery.ini]: Agreed

**Comment by szepty-ziemi** [comment]
This one seems to be ready to merge. Is it?

**Comment by Elfangor93** [comment]
I would like to merge PR #47 first. Otherwise this PR will produce again a lot of conflicts in #47.

**Comment by AlexanderSupp** [comment]
And of course, after PR #47, it's time to merge this small function also into main.

**Comment by Elfangor93** [comment]
I have tested this item ✅ successfully. Thank you very much.

**Pull request merged**